browser/src/control/Control.LokDialog.js

```diff
@@ -660,6 +660,12 @@ L.Control.LokDialog = L.Control.extend({
 		// Override default minHeight, which can be too large for thin dialogs.
 		L.DomUtil.setStyle(dialogContainer, 'minHeight', height + 'px');
 
+		// Title bar may overflow due to range name. So we should have max width
+		var titleBar = dialogContainer.previousSibling;
+		var leftPadding = window.getComputedStyle(titleBar).getPropertyValue('padding-left');
+		var rightPadding = window.getComputedStyle(titleBar).getPropertyValue('padding-right');
+		L.DomUtil.setStyle(titleBar, 'maxWidth', width - (leftPadding + rightPadding) + 'px');
```

Comment:
The problem is here, leftPadding and rightPadding are coming with `px` suffix

Reply:
so either you parse that before the calculation, which seems a bit ineffective since anyhow we end up adding the at the end. OR you simply set the style wrapped in `calc()` and let the css handle the calculation: so at the end the browser seems something like `style="max-width: calc(600px - (16px + 16px));"`

Reply:
so either you parse that before the calculation, which seems a bit ineffective since anyhow we end up adding the at the end. OR you simply set the style wrapped in calc() and let the css handle the calculation: so at the end the browser seems something like style="max-width: calc(600px - (16px + 16px));" `'calc(' + width + 'px ' + '- (' + leftPadding + ' + ' + rightPadding + '))');`

@eszkadev 
> Parsing will have non-visible impact (it's single operation). Such thing done thousand times per second would be a problem.
> I'm happy with both solutions. Correct paring or css calculation.
> we do 'px' removal in many places already...